Web/starter_project_barbies/pages/blog/BlogDetail.tsx

```diff
@@ -0,0 +1,137 @@
+import BlogCard from '@/components/blog/BlogCard'
+import fonts from '@/styles/blog/blog-detail.module.css'
+import { Blog } from '@/types/blog'
+import Image from 'next/image'
+
+interface BlogDetailProps {
+  blog: Blog
+}
+
+interface BlogAuthorDetailProps {
+  author: {
+    name: string
+    email: string
+    image: string
+    profession: string
+    userName: string
+  }
```

Comment:
for this definition, I think the Blog interface has its own author definition right?
you can use `author: Blog['author']` to refer to the author definition inside of the blog definition
this makes it so that there is only one place to modify the author definition (and avoid duplicate wirk)